server/src/main/java/io/spine/system/server/SystemFeatures.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2019, TeamDev. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.system.server;
+
+import com.google.errorprone.annotations.CanIgnoreReturnValue;
+import io.spine.annotation.Internal;
+
+/**
+ * System bounded context feature configuration.
+ */
+public final class SystemFeatures {
+
+    private boolean commandLog;
+    private boolean aggregateMirrors;
+    private boolean storeEvents;
+
+    /**
+     * Prevents direct instantiation.
+     */
+    private SystemFeatures() {
+    }
+
+    /**
+     * Obtains the default configuration.
+     *
+     * <p>By default, the system context:
+     * <ol>
+     *     <li>Stores Aggregate mirrors for querying.
```

Comment:
Let's speak of this as of `Enables querying of the latest {@code Aggregate} states.`

And I would name the `aggregateMirrors` field in the same manner.